lib/Form/Processor/ContentFormProcessor.php

```diff
@@ -119,7 +119,7 @@ public function processPublish(FormActionEvent $event)
 
         $draft = $this->saveDraft($data, $form->getConfig()->getOption('languageCode'));
         $versionInfo = $draft->versionInfo;
-        $content = $this->contentService->publishVersion($versionInfo, [$versionInfo->initialLanguageCode]);
+        $content = $this->contentService->publishVersion($versionInfo);
```

Comment:
This will most likely break [EZP-29694](https://jira.ez.no/browse/EZP-29694) fixed via #296. As stated in ezsystems/ezpublish-kernel#2846 the issue is probably caused by missing target there.

Reply:
issue fixed in the kernel https://github.com/ezsystems/ezpublish-kernel/pull/2858